CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+## 0.32.0 (2017-12-01)
+
+#### BREAKING CHANGES
+* remove `--no-mount` from `exo run` and `exo test`.
+  * `exo run` always mounts
+  * `exo test` never mounts
+* remove `exo create` in favor of `exo init` which initializes the current directory as an exosphere project
+* dependency volumes have been updated to use named docker volumes
+  ```yml
+  # Before
+  development:
+    dependencies:
+      - name: 'mongo'
+        version: '3.4.0'
+        config:
+          volumes:
+            - '{{EXO_DATA_PATH}}:/data/db'
+
+  # After
+  development:
+    dependencies:
+      - name: 'mongo'
+        version: '3.4.0'
+        config:
+          persist:
+            - /data/db
+  ```
+* update service environment variables from development/production to local/remote
+  ```yml
+  # Before
+  environment:
+    development:
+      ENV2: value2
+      ENV3: dev_value3
+    production:
+      ENV3: prod_value3
+
+  # After
+  environment:
+    local:
+      ENV2: value2
+      ENV3: dev_value3
+    remote:
+      ENV3: prod_value3
+  ```
+* terraform: inject dependency docker images as variables
+* update to terraform 0.11.0 and run terraform in a docker container
+* update cloudwatch alarm thresholds from 10/90 to 20/70
+
+#### New Features
+* add `exo generate` command which generates the docker compose and terraform files
+  * use `exo generate --check` in order to verify the files are up to date
+  * the files are also updated on each run of `exo run`, `exo clean`, `exo test`, and `exo deploy`
```

Comment:
@hugobho @alexdavid this may have a bad consequence. Now a feature branch may update the terraform file (which could slip through a review). The updates to the terraform file now become less deliberate since generate and all commands do it as well. `exo deploy --update-services` is a normal deploy aside from validating that the terraform file hasn't changed. I think we need to change `--update-services` to `--auto-approve`.